.github/workflows/wheels.yaml

```
@@ -28,11 +28,11 @@ jobs:
           - os: "ubuntu-20.04"
             os-name: "focal"
     env:
-      BUILD_COMMIT: 1.3.2
+      BUILD_COMMIT: maint-1.3
       PLAT: "${{ matrix.platform }}"
       MB_PYTHON_VERSION: "${{ matrix.python }}"
       MB_ML_VER: "2014"
-      BUILD_DEPENDS: "oldest-supported-numpy cython==0.29.28 wheel==0.34.2"
+      BUILD_DEPENDS: "oldest-supported-numpy cython>=0.29 wheel==0.34.2"
```

Latest Cython and wheel versions to ensure good Python 3.11 compatibility.
```suggestion
      BUILD_DEPENDS: "oldest-supported-numpy cython>=0.29.32 wheel>=0.37"
```